rundeckapp/grails-app/controllers/rundeck/controllers/StorageController.groovy

```diff
@@ -199,6 +199,11 @@ class StorageController extends ControllerBase{
             return renderError("unauthorized")
         }
         if((askedForContent || anyContent) && !maskContent) {
+            //check if download is enabled by feature flag for api public keys retrieval
+            if(!forceDownload && request.method == "GET" && !configurationService.getBoolean("feature.apiPublicKeysDownload.enabled", false)){
```

Comment:
I think could just do this:
```
boolean downloadEnabled = configurationService.getBoolean("feature.apiPublicKeysDownload.enabled", false)
if(downloadEnabled && (askedForContent || anyContent) && !maskContent) {
```

fyi, `forceDownload` just means force browser to download a file by using content-disposition header.

Reply:
Based [on this test](https://app.circleci.com/pipelines/github/rundeck/rundeck/17326/workflows/21a55519-8147-4bdb-9d1c-1d60b9fe04b2/jobs/200038/tests), it is also called for POST request. Should this feature flag block for POST too?

Reply:
@carlosrfranco yes, any request method